components/hybrid_engine/src/misc.rs

```diff
@@ -35,6 +35,10 @@ where
         strategy: engine_traits::DeleteStrategy,
         ranges: &[engine_traits::Range<'_>],
     ) -> Result<bool> {
+        for r in ranges {
+            self.region_cache_engine()
+                .evict_range(&CacheRange::new(r.start_key.to_vec(), r.end_key.to_vec()));
```

Comment:
I think the `delete_ranges_cf` can be called multiple times for each range(region) since there are `write cf`, `default cf` and `lock cf`. And the evict_range may be called multiple times for each range.

Reply:
Yes.